### DISCUSSION TITLE
How is `svg-sprite-loader` configured in rsbuild?

### DISCUSSION THREAD

**Discussion by coderwyd** [web-infra-dev/rsbuild#1700] [General]
I'm currently having issues migrating from vue-cli to rsbuild. Now the SvgIcon component displays blank, what do I need to configure to make it display normally.

SvgIcon.vue
`
<svg
    class="svg-icon inline-block overflow-hidden fill-current"
    aria-hidden="true"
    :class="[
      className,
      $attrs.class,
      disabled ? 'text-text-color-2 cursor-not-allowed' : '',
    ]"
    :style="getStyle"
    v-on="$listeners">
<use :xlink:href="symbolId" />
</svg>
`

**Comment by wenqian0812** [comment]
https://www.npmjs.com/package/rsbuild-svg-sprite-loader
